### DISCUSSION TITLE
What’s the difference between sockpuppet and htmx?

### DISCUSSION THREAD

**Discussion by simkimsia** [jonathan-s/django-sockpuppet#76] [Q&A] [answered]
I'm a bit lost after reading the documentation.

How is this different from htmx? As in [https://htmx.org](htmx.org)

**Comment by jonathan-s** [comment, marked as answer]
@simkimsia, I'd say the ideas are very similar to each other it's only the implementation that is different. (see https://en.wikipedia.org/wiki/Multiple_discovery). If you're happy using htmx, just continue using that, from the outlook it looks pretty solid.
